app/controllers/application_controller.rb

```diff
@@ -37,4 +37,10 @@ def check_entitled(entitlement)
 
     required_entitlements.map { |e| entitlement.send(e) }.all?
   end
+
+  def instance_link(instance)
```

Comment:
Not sure this should be here either. Possibly in a concern or a tagging mixin / service?

Reply:
I'm just gonna move it to the tag controller since that is the only place that uses it - and this will be in the common gem eventually.